adiachenko/src/components/FilmForm/index.tsx

```diff
@@ -0,0 +1,214 @@
+import React, {SyntheticEvent, useState} from "react";
+import Select from 'react-select';
+import { ThemeProvider } from "styled-components";
+import { Button, ButtonTheme } from "../Button";
+import { Form } from "./style";
+
+type initialFilmType = {
+    id?: number,
+    title: string,
+    release_date: string,
+    genres: string[],
+    overview: string,
+    poster_path: string,
+    runtime: number
+}
+
+type FilmFormProps = {
+    film?: initialFilmType,
+    edit?: boolean,
+    closeForm: () => void
+};
+
+const initialFilm: initialFilmType = {
+    title: '',
+    release_date: '',
+    genres: [],
+    overview: '',
+    poster_path: '',
+    runtime: 0
+}
+
+const options = [
```

Comment:
I didn't quite get the purpose of having 2 properties with exact same values. However, you may consider employing TypeScript enums:
```typescript
enum options {
   Fantasy = 'Fantasy'
   ..
}
```

Reply:
To cover multiselect was desided to use 'react-select' npm package.
We need to use 2 same values becouse Select component from 'react-select' accept for each option value and label (and doesnt wark with only label/value or simple string)
Enum added